src/main/java/com/openlattice/hazelcast/serializers/PropertyTypeStreamSerializer.java

```diff
@@ -74,6 +74,7 @@ public static void serialize( ObjectDataOutput out, PropertyType object ) throws
         out.writeBoolean( object.isPii() );
         out.writeInt( object.getAnalyzer().ordinal() );
         out.writeInt( object.getPostgresIndexType().ordinal() );
+        SetStreamSerializers.fastStringSetSerialize( out, object.getEnumValues() );
```

Comment:
EnumValues is optional, right? I think you need an optional stream serializer method here

Reply:
At this point, enumValues are not optional, but a `LinkedHashSet`, so I think this is the right serializer to use.